common/lib/xmodule/xmodule/course_module.py

```diff
@@ -197,10 +197,11 @@ class CourseFields(object):
         scope=Scope.settings,
     )
     advertised_start = String(
-        display_name=_("Course Advertised Start Date"),
+        display_name=_("Course Advertised Start"),
         help=_(
-            "Enter the date you want to advertise as the course start date, if this date is different from the set "
-            "start date. To advertise the set start date, enter null."
+            "Enter an optional advertised starting time frame for the course."
+            "For example, if you wish to advertise \'Fall 2025\' or \'Later this year\', enter it here."
+            "To use the start date that you have set, enter null for the value."
```

Comment:
@yro @sstack22 Here is my suggestion:
"Enter the text that you want to use as the advertised starting time frame for the course, such as “Winter 2018”. If you enter null for this value, the start date that you have set for this course is used."